portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.utils.CommentDataStore;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles deleting comment content. */
+@WebServlet("/delete-comment")
+public final class DeleteCommentServlet extends HttpServlet {
+    private static final String KIND = "Comment";
+    // TODO: actually this constant is used in both Comment and DeleteCommentServlet,
+    //  seems not proper to put it private static final, how i shall do it?
+
+    private CommentDataStore commentDataStore;
+
+    @Override
+    public void init() {
+        this.commentDataStore = new CommentDataStore(DatastoreServiceFactory.getDatastoreService());
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Should you handle delete item in delete method? do they have doDelete? Normally post method is used to create item.

Reply:
From the tutorial
![image](https://user-images.githubusercontent.com/49228945/84466635-68c03200-acac-11ea-9f5f-be6fd1e16a79.png)

Well it does have `doDelete()`